<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>1500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mod Manager Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="gamesScrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>200</width>
      <height>795</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>198</width>
       <height>793</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="gamesVLayout">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="createGameButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>70</y>
      <width>90</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>–î–æ–±–∞–≤–∏—Ç—å –∏–≥—Ä—É</string>
    </property>
   </widget>
   <widget class="QLabel" name="enterTitleLabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>22</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>–í–≤–æ–¥ –Ω–∞–∑–≤–∞–Ω–∏—è –∏–≥—Ä—ã</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gameLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>–í–≤–µ–¥–∏—Ç–µ –Ω–∞–∑–≤–∞–Ω–∏–µ –º–æ–¥–∞</string>
    </property>
    <property name="maxLength">
     <number>40</number>
    </property>
   </widget>
   <widget class="QPushButton" name="renameGameButton">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>70</y>
      <width>30</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>üñâ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nameGameButton">
    <property name="geometry">
     <rect>
      <x>146</x>
      <y>70</y>
      <width>30</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>‚úî</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteGameButton">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>70</y>
      <width>30</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>üóë</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addBlankModButton">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>30</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>–ë–µ–∑ –¥–∞–Ω–Ω—ã—Ö</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameModsLabel">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>80</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>–ò–≥—Ä–∞:</string>
    </property>
   </widget>
   <widget class="QLabel" name="addModInfo1">
    <property name="geometry">
     <rect>
      <x>443</x>
      <y>5</y>
      <width>135</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>–î–æ–±–∞–≤–∏—Ç—å –º–æ–¥–∏—Ñ–∏–∫–∞—Ü–∏—é</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addModWInfoButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>–î–∞–Ω–Ω—ã–µ –∏–∑ —Ñ–∞–π–ª–∞</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameModsNumberLabel">
    <property name="geometry">
     <rect>
      <x>443</x>
      <y>80</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>–ö–æ–ª–∏—á–µ—Å—Ç–≤–æ –º–æ–¥–æ–≤:</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>561</width>
      <height>87</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="filtrationLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="toHighlightLabel">
       <property name="text">
        <string>–í—ã–¥–µ–ª–∏—Ç—å</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="requiredButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>–ü–æ —Ç—Ä–µ–±–æ–≤–∞–Ω–∏—é/—Å–æ–≤–º–µ—Å—Ç–∏–º–æ—Å—Ç–∏</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="tagsEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>–í–≤–µ–¥–∏—Ç–µ —Ç—ç–≥/—Ç—ç–≥–∏</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="gameVersionButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>–ü–æ –≤–µ—Ä—Å–∏–∏ –∏–≥—Ä—ã</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="supVersionButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>–ü–æ –ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º–æ–π –≤–µ—Ä—Å–∏–∏</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="gameNameEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>–í–≤–µ–¥–∏—Ç–µ –Ω–∞–∑–≤–∞–Ω–∏–µ –º–æ–¥–∞</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="tagsButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>–ü–æ —Ç—ç–≥—É/—Ç—ç–≥–∞–º</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="ABCButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>–í –∞–ª—Ñ–∞–≤–∏—Ç–Ω–æ–º –ø–æ—Ä—è–¥–∫–µ</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="toFiltrateLabel">
       <property name="text">
        <string>–§–∏–ª—å—Ç—Ä–æ–≤–∞—Ç—å </string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="gameVersionEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>–í–≤–µ–¥–∏—Ç–µ –≤–µ—Ä—Å–∏—é –∏–≥—Ä—ã</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="saveAllMods">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>–°–æ—Ö—Ä–∞–Ω–∏—Ç—å –≤—Å–µ –º–æ–¥—ã</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="updateAllMods">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>–û–±–Ω–æ–≤–∏—Ç—å –≤—Å–µ –º–æ–¥—ã</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="modsScrollArea">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>100</y>
      <width>990</width>
      <height>795</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>988</width>
       <height>793</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="modsVLayout">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTextEdit" name="informationText">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>2</y>
      <width>165</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0)</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>–ó–¥–µ—Å—å –±—É–¥–µ—Ç –≤—ã–≤–æ–¥–∏—Ç—Å—è –≤—Å—è –∏–Ω—Ñ–æ—Ä–º–∞—Ü–∏—è –ø—Ä–∏ –æ—à–∏–±–∫–∞—Ö –∏–ª–∏ –∑–∞–º–µ—á–∞–Ω–∏—è—Ö</string>
    </property>
   </widget>
   <widget class="QLabel" name="languageLabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>2</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>–Ø–∑—ã–∫ | Lang:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RULang">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>2</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RU</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="ENLang">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>2</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>EN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="allFolderModsButton">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>58</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>–î–æ–±–∞–≤–∏—Ç—å –≤—Å–µ –º–æ–¥—ã –∏–∑ –ø–∞–ø–∫–∏</string>
    </property>
   </widget>
   <widget class="QLabel" name="savedModsNumberLabel">
    <property name="geometry">
     <rect>
      <x>641</x>
      <y>80</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>–ú–æ–¥–æ–≤ –≤ –±–∞–∑–µ –¥–∞–Ω–Ω—ã—Ö:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
